store/store.go

```diff
@@ -2204,7 +2204,9 @@ func (s *Store) snapAction(ctx context.Context, currentSnaps []*CurrentSnap, act
 	}
 
 	installNum := 0
+	downloadNum := 0
```

Comment:
I see tests for the error struct, but not for the new code here? missing something?